packages/nextjs/app/builders/0x7b32DF263625bFCC9EdF743A382b81c2e1A567Ec/page.tsx

```diff
@@ -0,0 +1,197 @@
+              <h2 className="text-sm uppercase tracking-wider text-zinc-500 dark:text-zinc-400">Snapshot</h2>
+              <ul className="mt-3 space-y-2 text-sm">
+                <li className="flex items-center justify-between">
+                  <span className="text-zinc-600 dark:text-zinc-400">Primary Stack</span>
+                  <span className="font-medium">Foundry · Next.js</span>
+                </li>
+                <li className="flex items-center justify-between">
+                  <span className="text-zinc-600 dark:text-zinc-400">EVM Focus</span>
+                  <span className="font-medium">Security · Gas</span>
+                </li>
+                <li className="flex items-center justify-between">
+                  <span className="text-zinc-600 dark:text-zinc-400">Timezone</span>
+                  <span className="font-medium">IST</span>
+                </li>
+              </ul>
+            </div>
+
+            <div className="rounded-2xl border border-zinc-200 dark:border-zinc-800 bg-white/70 dark:bg-zinc-900/60 backdrop-blur p-5 shadow-sm">
+              <h2 className="text-sm uppercase tracking-wider text-zinc-500 dark:text-zinc-400">Tech. Stack</h2>
+              <div className="mt-3 flex flex-wrap gap-2">
+                <span className="text-xs px-2 py-1 rounded-md bg-zinc-100 dark:bg-zinc-800 border border-zinc-200 dark:border-zinc-700">
+                  Solidity
+                </span>
+                <span className="text-xs px-2 py-1 rounded-md bg-zinc-100 dark:bg-zinc-800 border border-zinc-200 dark:border-zinc-700">
+                  Foundry
+                </span>
+                <span className="text-xs px-2 py-1 rounded-md bg-zinc-100 dark:bg-zinc-800 border border-zinc-200 dark:border-zinc-700">
+                  Hardhat
+                </span>
+                <span className="text-xs px-2 py-1 rounded-md bg-zinc-100 dark:bg-zinc-800 border border-zinc-200 dark:border-zinc-700">
+                  Huff
+                </span>
+                <span className="text-xs px-2 py-1 rounded-md bg-zinc-100 dark:bg-zinc-800 border border-zinc-200 dark:border-zinc-700">
+                  Yul
+                </span>
+                <span className="text-xs px-2 py-1 rounded-md bg-zinc-100 dark:bg-zinc-800 border border-zinc-200 dark:border-zinc-700">
+                  TypeScript
+                </span>
+                <span className="text-xs px-2 py-1 rounded-md bg-zinc-100 dark:bg-zinc-800 border border-zinc-200 dark:border-zinc-700">
+                  Next.js
+                </span>
+                <span className="text-xs px-2 py-1 rounded-md bg-zinc-100 dark:bg-zinc-800 border border-zinc-200 dark:border-zinc-700">
+                  Python
+                </span>
+                <span className="text-xs px-2 py-1 rounded-md bg-zinc-100 dark:bg-zinc-800 border border-zinc-200 dark:border-zinc-700">
+                  Go
+                </span>
```

Comment:
Consider mapping over a list of link data instead of repeating the JSX for each one. It'll make the code cleaner and easier to maintain.